lib/addons/kuberay/index.ts

```diff
@@ -0,0 +1,55 @@
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface KubeRayAddOnProps extends HelmAddOnUserProps {
+    /**
+     * To Create Namespace using CDK
+     */    
+    createNamespace?: boolean;
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+const defaultProps: HelmAddOnProps & KubeRayAddOnProps = {
+  name: "kuberay-operator",
+  chart: "kuberay-operator",
+  namespace: "default",
+  version: "1.0.0-rc.0",
```

Comment:
Why use an RC version for the release? I see the have 1.0.0 - let's use that. We generally avoid using non-GA versions as default for the addon (with rare exceptions).